db/install.xml

```diff
@@ -79,5 +79,16 @@
         <INDEX NAME="courseid" UNIQUE="false" FIELDS="courseid"/>
       </INDEXES>
     </TABLE>
+    <TABLE NAME="block_xp_default_filters" COMMENT="Default Filters">
```

Comment:
As stated above, maybe we do not want another table. I'm still uncertain at this stage though.